pinot-common/src/thrift/request.thrift

```diff
@@ -125,6 +125,7 @@ struct BrokerRequest {
  10: optional string bucketHashKey;
  11: optional bool enableTrace;
  12: optional string responseFormat;
+ 13: optional list<string> routingOptions;
```

Comment:
`enableTrace` hasn't been modified in this, though we'll have the flexibility to do so in the future and give it parameters (eg. trace only the aggregation time, all operators or broker routing time, etc.).